yb-voyager/src/utils/utils.go

```diff
@@ -773,25 +773,57 @@ func MatchesFormatString(format, final string) (bool, error) {
 	return re.MatchString(final), nil
 }
 
-// FormatToRegex converts a format string with %s placeholders to a regex pattern.
-// It assumes(and support) that the only format specifier is %s.
-func FormatToRegex(format string) (string, error) {
-	// Split the format string at each %s to replace that with regex capture group.
-	parts := strings.Split(format, "%s")
+// formatToRegex converts a format string containing %s and %v
+// into a regex pattern.
+//   - %s => (.+?)    (capturing group)
+//   - %v => (?:.+?)  (non-capturing group)
```

Comment:
wouldn't the existing implementation work? (since you were only checking for %s? Why even define a group for %v?